modular_skyrat/modules/borg_buffs/code/robot.dm

```diff
@@ -116,3 +116,78 @@
 		tool_behaviour = TOOL_KNIFE
 		to_chat(user, span_notice("You attach the knife bit to [src]."))
 		icon_state = "knife_screw_cyborg"
+
+/obj/item/inducer/cyborg
+	name = "Cyborg Inducer"
+	desc = "A tool for inductively charging internal power cells using the battery of a cyborg"
+	powertransfer = 250
+
+
+
+/obj/item/inducer/cyborg/attackby(obj/item/Weapon, mob/user)
+	if(Weapon.tool_behaviour == TOOL_SCREWDRIVER)
+		return
+	if(istype(Weapon, /obj/item/stock_parts/cell))
+		return
```

Comment:
No need to make any checks, we don't want anything to happen with it. Also, lower case for variable names.
```suggestion
/obj/item/inducer/cyborg/attackby(obj/item/weapon, mob/user)
	return
```